onadata/libs/utils/logger_tools.py

```diff
@@ -275,6 +280,37 @@ def check_submission_permissions(request, xform):
               }))
 
 
+def check_submission_encryption(xform: XForm, xml: bytes) -> NoReturn:
+    """
+    Check that the submission is encrypted or unencrypted depending on the
+    encryption status of an XForm.
+
+    The submission is invalid if the XForm's encryption status is different
+    from the submissions
+    """
+    encryption_status = False
+    submission_element = ET.fromstring(xml)
+    encrypted_attrib = submission_element.attrib.get('encrypted')
+    required_encryption_elems = [
+        elem.tag for elem in submission_element
+        if elem.tag in REQUIRED_ENCRYPTED_FILE_ELEMENTS]
+
+    # Check the validity of the submission
+    if encrypted_attrib == "yes" or len(required_encryption_elems) > 1:
+        if (not len(required_encryption_elems) == 2 or
```

Comment:
The expression `len(required_encryption_elems)` is used more than once here, can we assign it to variable and use the variable instead?